### ISSUE TITLE
Update installation instructions

### ISSUE THREAD

**Issue by mondeja** [gaucho-labs/leptos-hotkeys#51]
This is something that is not intuitive but very seems very important when installing `leptos` libraries. Leptos allows you to build applications that are generated, at least:

- On client side with...
   - Client side rendering (`csr` feature): This means that the component is built on the browser.
   - Hydration (`hydrate` feature): The component is rendered first on the server (`ssr`) and then is hydrated on the client, so it runs 2 times.
- On server side (`ssr` feature).

So, in an SSR + hydrate application, we build two versions of the code: to render on the server and to hydrate on the client. Because of that, we need to allow all features to be optionally enabled to make sure that the application can be built for server, hydrated and rendered on client side.

See ["Hydration bugs" chapter of the Leptos book ](https://book.leptos.dev/ssr/24_hydration_bugs.html#a-thought-experiment):

> Each time you reload the page, where do I run? should log once on the server and once on the client.

Following that seems that almost all Leptos libraries should recommend to install for all contexts **from the beginning**. So the installation instructions should be something like:

```toml
[dependencies]
leptos_hotkeys = "0.1.6"

[features]
csr = ["leptos_hotkeys/csr"]
hydrate = ["leptos_hotkeys/hydrate"]
ssr = ["leptos_hotkeys/ssr"]
```

This is what [leptos-use recommends for SSR](https://leptos-use.rs/server_side_rendering.html), with the difference that `leptos-use` does not have an `csr` feature. If a library has a `csr` feature because needs mutually exclusive functionalities or because it prefers to explicitly obligate app consumers to enable the `csr` feature, it should recommend the features approach from the beginning. That's what I'm doing in [`leptos-fluent`](https://leptos-use.rs/server_side_rendering.html).

**Comment by mondeja** [comment]
Thoughs about this @friendlymatthew?

**Comment by friendlymatthew** [comment]
> Thoughs about this @friendlymatthew?

Yeah this was definitely a todo. Funny enough, I was chatting with Mr. Leptos-use about this today.

Completely agree this is needed

**Comment by friendlymatthew** [comment]
We will probably have to clean up the CFG imports.

'''

It seems you really don't need the hydrate or csr features. You can safely have web-sys and wasm-bindgen as dependencies on the server as long as you don't try and instantiate sth from them

'''

**Comment by friendlymatthew** [comment]
Correct me if I'm wrong but I think this is fixed? Via #63 and #64
